synth.py

```diff
@@ -152,18 +185,25 @@
 templates = common_templates.node_library(
     source_location="build/src", test_project="node-gcloud-ci"
 )
-
 s.copy(templates, excludes=[".eslintrc.json", ".kokoro/**/*", ".github/CODEOWNERS"])
 
 # Remove auto-generated packaging tests
 os.system('rm -rf dev/system-test/fixtures dev/system-test/install.ts')
 
-node.install()
-node.fix()
+# Compile protos and unlink files before moving them since compile_protos()
+# requires that the protos under the same directory as src/.
 os.chdir("dev")
 node.compile_protos()
 os.chdir("protos")
 os.unlink('protos.js')
 os.unlink('protos.d.ts')
 subprocess.run('./update.sh', shell=True)
+
+# Copy the protos directory into dev/src after compiling.
 os.chdir("../../")
+s.move("dev/protos", "dev/src/protos")
+os.system("rm -rf dev/protos")
+
```

Comment:
I think it might make sense to update the proto location in the generated client here. Before you move them, the source is not valid, is it?

Reply:
Moved.